from .error import AppException


class NotFoundException(AppException):
    pass
